Instanssi/main2016/static/main2016/css/style.scss

```diff
@@ -1,5 +1,5 @@
 @import 'reset.scss';
-@import 'base_layout/css/common.scss';
+@import 'base_layout/css/common.2017.scss';
```

Comment:
main2016 -> common.2016.scss ?

Reply:
There is no common.2016.scss :)

This probably happened by accident in 2017, but the 2016 and 2017 sites have a pretty similar color palette and don't really use SCSS all that much, so they can work with the same base_layout variables. The later versions of the theme use more variables and stuff so they have their own common style files.

Added a note about this into the file.